data/json/recipes/other/materials.json

```diff
@@ -32,8 +32,7 @@
       [ [ "forge", 80 ], [ "char_forge", 80 ] ]
     ],
     "components": [
```

Comment:
Will have to later on when I get the chance, and might clean up a few assorted recipe files, if I can determine that they are not currently the subject of a pull request. I suspect that the risk of multiple merge conflicts is the reason there has not been a large-scale cleanup of recipe formatting.